### ISSUE TITLE
Radio streaming don't work anymore after update

### ISSUE THREAD

**Issue by Gnomsen** [bromite/bromite#71]
Since last update of Bromite v67.0.3396.82 to v67.0.3396.88, radiostreaming does not play sound.

https://www.psychedelik.com/ambient.html
Bromite v67.0.3396.82 works well
Bromite v67.0.3396.88 no sound anymore

Chromium v67.0.3396.82 & chromium v67.0.3396.88 are able to play radiostreams with sound

Maybe it has something to do with this entry found in the latest changelog:

"use Android Media Player for URLs"

**Comment by csagan5** [comment]
That entry you mentioned it only matters for downloaded files, as instead of the Android-level app it allows the browser to open media files directly.

I will check if there is any hint in the Javascript console.

**Comment by csagan5** [comment]
The console contains:
```
Mixed Content: The page at 'https://www.psychedelik.com/ambient.html' was loaded over HTTPS, but requested an insecure audio file 'http://195.154.166.226:8002/stream/1/;'. This content should also be served over HTTPS.
```

Since you tested with Chromium then it must be due to some patch...somebody should go through all the patches to find out which one it is.

**Comment by csagan5** [comment]
I think you were right, removing that patch solves the issue; in next release that patch will be dropped (for other reasons) so this bug should be fixed too.

**Comment by Gnomsen** [comment]
In advance Thanks alot for the fix and for the project itself!

**Comment by csagan5** [comment]
Fixed in 67.0.3396.107
